inference.py

```diff
@@ -9,15 +10,15 @@
 from segment import Segment
 from timer import Timer
 
-FEET_PER_MILE = 5280
 STOP_PROXIMITY = 150
-SCORE_THRESHOLD = 50
+SCORE_THRESHOLD = 7
```

Comment:
I know almost nothing about score threshold but this seems low. Is the idea that a very low score is acceptable in some cases?

Reply:
The training data has allowed me to just play with different values for settings, and it turns out that a lower value here gives more accurate results overall.